internal/storage/bucket_handle.go

```diff
@@ -133,8 +133,17 @@ func (bh *bucketHandle) CreateObject(ctx context.Context, req *gcs.CreateObjectR
 	// We can't use defer to close the writer, because we need to close the
 	// writer successfully before calling Attrs() method of writer.
 	if err = wc.Close(); err != nil {
-		err = fmt.Errorf("error in closing writer: %v", err)
-		return
+		if err != nil {
+			switch ee := err.(type) {
```

Comment:
do you need a switch condition for one case or can you use if-else as well?

Reply:
fixed